cmd/access.go

```diff
@@ -60,8 +60,8 @@ type AccessOptions struct {
 
 // Complete adds any missing configuration that can only be added just before calling Run
 func (o *AccessOptions) Complete(f Factory, args []string) (err error) {
-	o.Instance = f.Instance()
-	return nil
+	o.Instance, err = f.Instance()
+	return
```

Comment:
can drop the return here

Reply:
always use an explicit return value. It's fine to use named return values as documentation, but not as default returns